### ISSUE TITLE
Training details&pretrained models

### ISSUE THREAD

**Issue by rayray9999** [wengflow/robust-e-nerf#3]
Thank you for your great work! I'm currently experimenting with the models and need some help regarding the specifics of training.

I've been training the ficus/easy scene using the default settings in synthetic.yaml, with only the parameters suggested in the paper modified. It took me about a day to reach 3600 iterations on a single RTX 4090.

I attempted to increase the train_eff_ray_sample_batch_size to 2^23 and decrease limit_train_batches to 125, but the reconstructed scenes ended up blank. I also tried using two 4090s with num_workers_per_node set to 64, but it didn't significantly speed up the training time.

Could you let me know how long it takes for you to complete the full training process (40000 iterations) for successfully reconstructed scenes? What did you do for accelerating? And have I made any mistakes with my modifications?

Additionally, I wanted to ask if you plan on releasing pretrained models for some of the scenes used in your experiments?

**Comment by wengflow** [comment]
Hi, thanks for the interest.
1. You mentioned you are running on a RTX4090. Note that this GPU is only supported with CUDA >=11.8 (and thus PyTorch >=2). Hence, I believe that might be the culprit.
2. It should take ~1-1.5 hours to train the `synthetic.yaml` configuration on a RTX 4090, without any changes.
3. You mentioned:
    > with only the parameters suggested in the paper modified

    What specific parameters did you modify, apart from `data.dataset_directory`, `model.contrast_threshold.freeze`, `model.refractory_period.freeze` and/or `loss.weight.log_intensity_grad`, as pointed out in the README?
4. There is no plan to release the pretrained models, as replication of our results should (by right) be rather simple and straight-forward with our released dataset/simulator and training/evaluation code.

**Comment by rayray9999** [comment]
Hi, Thanks for your response and kindly help!
1.
I tried to build a CUDA 11.8 environment but the speed is still the same (one and a half minutes per epoch),and torch-batch-svd doesn't work in this environment.
So I use `cuda 11.7` and related cudatoolkit 
pytorch=1.13.0+cu117 
torchvision=0.14.0+cu117,
which can work well with torch-batch-svd.

Another Userwarning I get is 

> tinycudann was built for lower compute capability (86) than the system's (89). Performance may be suboptimal.

I'm not sure wether this will lower the speed immensely.

3.
I changed `data.num_workers_per_node: 32` as the system recommended
`model.eval_save_pred_intensity_img: true `
`lr_scheduler.multi_step_lr.milestones: [ 20000, 30000, 36000 ] 
`
`trainer.max_epochs: 40000`
`trainer.check_val_every_n_epoch:100`

I change the epoch as paper mentioned in Implementation Details.

**Comment by wengflow** [comment]
1. 1.5 minutes per epoch is about right. You can ignore the warning about `torch-batch-svd`, PyTorch native `torch.svd` is just fine. For the compute capability warning, it should not significantly affect our code, but I suspect that may be due to having multiple GPUs with differing compute capabilities on your computer (Refer to this [issue](https://github.com/NVlabs/tiny-cuda-nn/issues/341)).
2. Ignore the warning and keep `data.num_workers_per_node: 0`. Keep `trainer.max_epochs: 40`, `lr_scheduler.multi_step_lr.milestones: [ 20, 30, 36 ]` and `trainer.check_val_every_n_epoch: 1`. Note that 1 epoch = 1000 iterations/batches, as `limit_train_batches: 1000`.

**Comment by rayray9999** [comment]
It works! Much appreciated!
